proposals/0569-multiline-strings.rst

```diff
@@ -112,6 +112,7 @@ A working prototype is available at `brandonchinn178/string-syntax <https://gith
 
       * See the "Common whitespace prefix calculation" section below for the specification of the calculation
       * If a line only contains whitespace, remove all of the whitespace
+      * Don't remove common whitespace immediately after the delimiter, see the example in *Section 3.2 Ignore leading characters*
```

Comment:
This isn't a change in the spec, it's a clarification that follows from the "Ignore leading characters" section:

> This means that characters immediately after the ``"""`` delimiter will be included verbatim